src/filters.js

```diff
@@ -358,7 +358,25 @@ var filters = {
 
         return res;
     },
-
+    
+    sum: function(arr, start, attr) {
+        if(typeof start !== 'number'){
+            start = 0;
+        }
+        
+        if(attr) {
+            arr = lib.map(arr, function(v) {
+                return v[attr];
+            });
+        }
+        
+        for(var i = 0; i < arr.length; i++) {
+            sum += arr[i];
```

Comment:
This var seems to be undeclared. And also you aren't using the `start` value anywhere. Were those intended to be the same var?